airflow/providers/apache/spark/example_dags/example_spark_dag.py

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG to submit Apache Spark applications using
+`SparkSubmitOperator`, `SparkJDBCOperator` and `SparkSqlOperator`.
+"""
+from airflow.contrib.operators.spark_jdbc_operator import SparkJDBCOperator
+from airflow.contrib.operators.spark_sql_operator import SparkSqlOperator
+from airflow.contrib.operators.spark_submit_operator import SparkSubmitOperator
```

Comment:
As an example, should we use `airflow.providers.apache.spark.operators` instead of deprecation one?

Reply:
Thank you @zhongjiajie, fixed.